### PR TITLE
Corrected Typos in CDNetworks ECP API (1.0)

### PR DESCRIPTION
**Pull request by marcfishelquantil**
I corrected typos in CDNetworks ECP API (1.0).

### CODE REVIEW

File: docs/portal/applications/using-advanced-ecp-features.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+    healthCheckTimeout: 10 # 10 by default. Scope: 1-50 in seconds
+    unhealthyThreshold: 3 # 3 by default. Must be a positive integer
+    healthCheckInterval: 1 # 1 by default. Scope: 1-5  in seconds
+    listenerPort: 80
+    name: listener-80
+    protocol: tcp
+    scheduler: lc
+```
+
+## Using Storage Resources
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/using-advanced-ecp-features.md:177]: Please help revise this section to remove anything related to persist-ssd. This storageclass has limited availability, and it is preferred not to show in documentation.

File: docs/portal/applications/updating-an-application.md [hunk 2]
```diff
@@ -1,24 +1,66 @@
-# Updating an Application
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/updating-an-application.md:1]: To update an application, there are 2 options, using the quick updates or using "Edit YAML". I do not see documentation about the second option, please include it.

File: docs/portal/applications/updating-an-application.md [hunk 3]
```diff
@@ -1,24 +1,66 @@
...
+# Quick Updating an Application
 
 The following procedure describes how to update applications from the Applications page. You can also update applications from the form used to [view application information](</docs/portal/applications/viewing-application-information.md>).
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/updating-an-application.md:3]: This paragraph shall be changed. Previously, we demoed how to update application from the Applications page, but now the demos are moved to the application information page.

File: docs/portal/applications/updating-an-application.md [hunk 4]
```diff
@@ -1,24 +1,66 @@
...
 
 1. In the left pane, click **Applications**.
-2. Find the application you want to update, then click its **Actions** menu and select **Update**. An Update Application form similar to the following appears.
+2. Click the name of the application you want to update.
+3. At the top right of the form, click the **Update** button, and then select one of the following quick update options:<ul><li> **Redistribute** allows you to add, edit, and delete distribution targets.</ul></li><ul><li>**Set container image** allows you to update your pods to a newer application version.br></ul></li><ul><li>**Upgrade resources** allows you to upgrade the spec of your container instance to maintain performance and/or save costs.</ul></li>
 
 ![null](</docs/resources/images/applications/applications-update.png>)
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/updating-an-application.md:9]: The screenshot is to be replaced.

File: docs/portal/applications/updating-an-application.md [hunk 5]
```diff
@@ -1,24 +1,66 @@
-# Updating an Application
+# Quick Updating an Application
 
 The following procedure describes how to update applications from the Applications page. You can also update applications from the form used to [view application information](</docs/portal/applications/viewing-application-information.md>).
 
 1. In the left pane, click **Applications**.
-2. Find the application you want to update, then click its **Actions** menu and select **Update**. An Update Application form similar to the following appears.
+2. Click the name of the application you want to update.
+3. At the top right of the form, click the **Update** button, and then select one of the following quick update options:<ul><li> **Redistribute** allows you to add, edit, and delete distribution targets.</ul></li><ul><li>**Set container image** allows you to update your pods to a newer application version.br></ul></li><ul><li>**Upgrade resources** allows you to upgrade the spec of your container instance to maintain performance and/or save costs.</ul></li>
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/updating-an-application.md:7]: There are some unexpected characters at the end of the second list item, ".br>".

File: docs/portal/applications/updating-an-application.md [hunk 6]
```diff
@@ -1,24 +1,66 @@
...
 
 ![null](</docs/resources/images/applications/applications-update.png>)
 
-3. Complete the fields in the Update Application form. Required fields are denoted by an asterisk (\*).
+##Redistributing Your Application
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/updating-an-application.md:11]: Many headings on this page are not in correct format. Please check.

File: docs/portal/applications/updating-an-application.md [hunk 7]
```diff
@@ -1,24 +1,66 @@
...
 
-4. Click **Submit & Deploy** to update your application automatically across ECP edge locations.<br>
+**Note:** Before proceeding, observe the following behaviors:<ul><li>The redistribution method cannot be changed using the Quick Update feature.</ul></li><ul><li>The application deployment process may take up to a few minutes, depending on the size of container images to be pulled, the target locations, and the number of expected instances. Once the application is deployed, **Deployed** appears as its status on the Applications page. To check the status and other details, use the applications list view and detail view. After the application is deployed, you can retrieve instances in [list view](</docs/portal/applications/retrieving-instances-of-an-application.md#retrieving-instances-of-an-application-in-list-view>) or [map view](</docs/portal/applications/retrieving-instances-of-an-application.md#retrieving-instances-of-an-application-in-map-view>)</ul></li><ul><li>Scaling down or redistributing an application that uses persistent storage volumes might delete existing persistent volumes along with pod instances.li></ul></li>
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/updating-an-application.md:15]: There are unexpected characters at the end of the last list item, ".li>".

File: docs/portal/applications/updating-an-application.md [hunk 8]
```diff
@@ -1,24 +1,66 @@
...
+3. To remove a target, click the trash can icon at the right side of the target.
+4. When you finish, click **Redistribute**.
+
+###Applications Distributed per Pop
+
+If your application is distributed per PoP, a dialog box similar to the following appears.
+
+<<new_Redistribute-Your-Application-Distribute-per-Pop.png goes here>>
+
+1. Under **Targets**, accept or change the PoPs and replicas per PoP shown for existing PoPs.
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/updating-an-application.md:34]: I believe you intended "existing targets" rather than "existing PoPs".

File: docs/portal/applications/updating-an-application.md [hunk 9]
```diff
@@ -1,24 +1,66 @@
...
+**Note:** This operation might cause your existing pods to be deleted and recreated.
+
+1. To change the Docker image name, replace the name in the **Image** field. Typically, this name consists of docker image registry, image, and tag.
+<br><ul>- If you omit the registry, the system defaults to docker hub.</ul><ul>- If you omit the tag, the system defaults to "latest".</ul><ul>- If your entry is not valid, the image pulling and application deployment fails.</ul></br>
+2. To change the image pull secret, drag the <strong>Change Image Pull Secret</strong> control to the right. Then click in the field displaying the current secret and select an image pull secret.
+3. To add an image pull secret, drag the <strong>Change Image Pull Secret</strong> control to the right. Click in the field displaying the current secret and select <strong>+Add Image Pull Secret</strong>. Complete the fields in the Add ImagePullSecret dialog box, and then click <strong>Submit</strong>. <br>
+4. When you finish, click **Update**.
+   
+##Upgrading Resources
+If you select **Upgrade resources** from the Quick Update drop-down list, a Set Container Image dialog box similar to the following appears.
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/updating-an-application.md:53]: "Set Container Image" here isn't correct.

File: docs/portal/applications/updating-an-application.md [hunk 10]
```diff
@@ -1,24 +1,66 @@
...
+If you select **Upgrade resources** from the Quick Update drop-down list, a Set Container Image dialog box similar to the following appears. 
+
+<<new_Upgrade-Container-Spec-Dialog-Box.png goes here>>
+
+**Note:** This operation might cause your existing pods to be deleted and recreated.
+
+1. To change the number of CPU requests, enter a value in the **CPU** field. The value you enter is measured in cores. To specify millicores, add an **m** to the end of the value.
+   **Note:** The CPU value should not be lower than 50 millicores, and should not cause the total amount of CPU requested by a pod to exceed 24 cores.
+2. To change the number of memory requests, enter a value in the **Memory** field. The value you enter is measured in bytes. To specify a different unit of measurement, add one of the following to the end of the value: E, P, T, G, M, K, Ei, Pi, Ti, Gi, Mi, Ki.
+**Note:** The memory value should not be lower than 50MB, and should not cause the total amount of memory requested by a pod exceed 128GB.
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/updating-an-application.md:62]: "to" is missing before "exceed"

File: docs/portal/applications/adding-and-deploying-applications.md [hunk 11]
```diff
@@ -1,514 +0,0 @@
-# Adding and Deploying Applications Using a Wizard
```
Comment (Jianghai-Quantil) [inline docs/portal/applications/adding-and-deploying-applications.md:1]: I notice that you've deleted this page and splitted the contents into 4 separate pages. Please modify the "routes" on this file (docs/manifest.json) as well, otherwise the new menus won't appear on the documentation site. Please let me know if you need help in this.
I notice that there are many links that point to this deleted page, as you can see in my other comments. Please check and fix broken links. Note that I might not have spotted all broken links.

File: docs/portal/applications/viewing-application-information.md [hunk 12]
```diff
@@ -12,14 +12,14 @@ The following procedure describes how to view information about applications.
...
 | 3                        | Click this button to delete the application.                                                               |
-| 4                        | Click this button to display the Specifications dialog box, where the file can be viewed in YAML or JSON format, copied, and downloaded.                                                                |
+| 4                        | Click this button to make quick updates to the application without having to edit the corresponding YAML file. Choices are:<br><ul><li><strong>**Redistribute**</strong> = redistributes your application across ECP locations to maintain performance, save costs, or both. After an application is created, you cannot change the distribution method.<li><strong>**Set container image**</strong> = updates your pods to a newer application version. Performing this action might delete or recreate your existing pods.<li><strong>**Upgrade resources**</strong> = upgrades the spec of your container instance to maintain performance, save costs, or both. Performing this action might delete or recreate your existing pods.</br></li>                                                               |
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/viewing-application-information.md:17]: Corrections made in two places within this topic.

File: docs/portal/applications/viewing-application-information.md [hunk 13]
```diff
@@ -12,14 +12,14 @@ The following procedure describes how to view information about applications.
 | **Element Number**       | **Description**                               |
 | -------------------------|-----------------------------------------------| 
 | 1                        | Click this button to return to the Applications page.                                                                      |
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/viewing-application-information.md:14]: In addition to this screen shot, there are 11 other updated ones I captured on 7/22 that have not yet been incorporated. I have refrained from adding snap shots because the ones I capture are of low resolution. Now that I know that  Eleazar is the point of contact for generating high-resolution screen shots (we worked together this week to add refreshed CDN360 screen shots), I will work with him to add high-res screen shots to the ECP help.

File: docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+
+![null](</docs/resources/images/applications/applications.png>)
+
+3. When prompted, click **Use a wizard**.
+
+![null](</docs/resources/images/applications/applications-add.png>)
+
+4. Complete the configure fields in the first page of the wizard.
+
+![null](</docs/resources/images/applications/applications-add-wizard-configure.png>)
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md:14]: Please see comment above.

File: docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
...container name, docker image and tag, CPU and memory resources, environment variables, and volume mounts for the first container (Container0).<br>Use Advanced Settings to define the following parameters:<br><ul><li><strong>Command</strong> = entrypoint array in JSON or YAML format. If omitted, the docker image's ENTRYPOINT is used. Variable references $(VAR_NAME) are expanded using the container's environment.</li><li><strong>Args</strong> = arguments to the entrypoint in JSON or YAML format. If omitted, the docker image's CMD is used. Variable references are expanded using the container's environment.</li><li><strong>Image Pull Policy</strong> =  select from Always, Never, or IfNotPresent. If latest tag is specified, this setting defaults to Always; otherwise, it defaults to IfNotPresent</li><li><strong>Liveness Probe</strong> = periodic probe of container service readiness. If the probe fails, the container will be removed from service endpoints.</li><li><strong>Readiness Probe</strong> = periodic probe of container service readiness. If the probe fails, the container will be removed from service endpoints.</br></li>                                                               |
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md:24]: Fixed.

File: docs/portal/applications/updating-an-application.md (hunk 7 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:15]: Fixed.

File: docs/portal/applications/updating-an-application.md (hunk 8 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:34]: Fixed.

File: docs/portal/applications/updating-an-application.md (hunk 9 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:53]: Fixed.

File: docs/portal/applications/updating-an-application.md (hunk 10 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:62]: Fixed.

File: docs/portal/applications/adding-and-deploying-applications.md (hunk 11 above)
Comment (marcfishelquantil) [reply docs/portal/applications/adding-and-deploying-applications.md:1]: When I meet w/Eleazar about getting high-res screen shots, I will ask whether he knows how to do this. If he doesn't, I will get back to you, Neil. Thank you.

File: docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md [hunk 16]
```diff
@@ -0,0 +1,76 @@
...
+
+**Note:** You can add containers, environment variables, and volume mounts by clicking their **\+ Add** links in the **Container** section.
+
+5. At the top right of the wizard, click **\> Next**. 
+6. Select whether to expose your application using a [Layer 4 load balancer](<#exposing-via-a-load-balancer>) or the[ public network](<#exposing-via-the-public-network>), and then complete the remaining fields based on your selection.
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md:29]: Text changed. Let's touch base about linked content.

File: docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+| Listeners                                                                                                                                                  | Complete the following fields:<br><ul><li><strong>Name</strong> = enter the name of the listener. </li><li><strong>Listener Port</strong> = enter the number of the ports on which the layer 4 load balancer must listen.</li> </li><li><strong>Backend Port</strong> = read-only field that shows the number of the backend pod port that receives traffic forwarded by the layer 4 load balancer. This is the same value entered for the Listener Port.</li></li><li><strong>Protocol</strong> = select either TCP or UDP.</li> </li><li><strong>Scheduler</strong> = select the load-balancing algorithm used to make load-balancing decisions. Choices are Round Robin (*default*) or Least Connection.</li> </li><li><strong>Health Check</strong> = select whether periodic health checks should be performed on backend pods. The load balancer will remove a pod that fails the health check.</li>            |
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md:39]: Fixed.

File: docs/portal/applications/adding-and-deploying-applications-using-an-existing-configuration-file.md [hunk 18]
```diff
@@ -0,0 +1,21 @@
+# Adding and Deploying Applications Using an Existing Configuration File
+
+1. [Configure an application](<#configuring-applications>) outside of the portal.
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/adding-and-deploying-applications-using-an-existing-configuration-file.md:3]: Fixed

File: docs/portal/applications/configuring-applications.md [hunk 19]
```diff
@@ -0,0 +1,131 @@
+# Configuring Applications
+
+The first step in deploying your application to ECP clusters is to use the ECP-defined “Application” object. An ECP “Application” typically consists of a target, a Kubernetes workload controller, and a Layer 4 load balancer.
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/configuring-applications.md:3]: Minor rephrasing and added links.

File: docs/portal/applications/configuring-applications.md [hunk 20]
```diff
@@ -0,0 +1,131 @@
...
+    replicas: 5
+```
+
+## Kubernetes Workload Controller
+
+The Kubernetes workload controller controls the state of an application instance. It includes specifications for the container image, compute/storage resources request, container startup commands and args, etc. The workload controller is configured the same as the [native Kubernetes Controller](<https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/>).
+
+## Layer 4 Load Balancer
+
+To expose your application, we recommend you use a Layer 4 load balancer as a frontend for your pods, unless there are special requirements to expose the pods directly. Unlike native Kubernetes, which defines a Service object for service discovery and load balancing of an application, the ECP offers a self-defined `LoadBalancer4` object to help you expose your application and load balance traffic. For more information, click [here](<#using-layer-4-load-balancing>).
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/configuring-applications.md:46]: Fixed

File: docs/portal/applications/configuring-applications.md [hunk 21]
```diff
@@ -0,0 +1,131 @@
...
+- An “application name” parameter is required when an application configuration is to be deployed. The value of this parameter is used to fill the `metadata.name` fields of the target, workload controller, and `LoadBalancer4` objects. If you specify these `metadata.name` fields in the application configuration, make sure the values are consistent with the “application name” parameter.
+- The target type must be either `Location` or `RegionPreference`. Mixed usage of the two types is not allowed. Because some ECP locations/regions might not be enabled for you, contact CDNetworks' sales engineers if you want to enable certain locations or regions, or if you have trouble deploying your application to ECP locations/regions.
+- Not all Kubernetes workload Controllers are supported. We recommend you use a [StatefulSet](<https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/>) Controller to run applications that require persistent storage. Otherwise, use a [ReplicaSet](<https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/>) Controller.
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/configuring-applications.md:128]: Fixed.

File: docs/portal/applications/configuring-applications.md [hunk 22]
```diff
@@ -0,0 +1,131 @@
...
+- When configuring a workload Controller, leave the `spec.replicas` field empty because the equivalent is configured in the target object.
+- If you need persistent volumes for your application, you can use two storage classes: `local-ssd` and `persist-ssd`. For more information, see [Using Storage Resources](<#using-storage-resources>).
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/configuring-applications.md:130]: Fixed

File: docs/portal/applications/updating-an-application.md (hunk 2 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:1]: I added "Edit YAML" as the new step 3 in Updating an Application.

File: docs/portal/applications/updating-an-application.md (hunk 3 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:3]: Fixed.

File: docs/portal/applications/updating-an-application.md (hunk 5 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:7]: Fixed

File: docs/portal/applications/updating-an-application.md (hunk 4 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:9]: Replaced screen shot.

File: docs/portal/applications/updating-an-application.md (hunk 6 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:11]: I confirmed that the heading formats are correct:
Heading 1 = Updating an Application
Heading 2 = Redistributing Your Application
Heading 3 = Applications Distributed per Region
Heading 3 = Applications Distributed per Pop
Heading 2 = Setting the Container
Heading 2 = Upgrading Resources

File: docs/portal/applications/using-advanced-ecp-features.md [hunk 23]
```diff
@@ -0,0 +1,284 @@
+# Using Advanced ECP Features
+
+## Request Public IP Address for a Pod
+
+When a pod gets created with the native Kubernetes, the pod is assigned a private IP address that is accessible only from within the Kubernetes cluster. CDNetworks’ ECP extensions to Kubernetes allow you to request a public IP address for a pod and support IPv4 and IPv6 protocols.
+
+## Request Public IPv4 Address
+
+To obtain a public IPv4 address for your pod, add an annotation to the pod spec, as marked in blue in the following YAML file example.
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/using-advanced-ecp-features.md:9]: Fixed.

File: docs/portal/applications/using-advanced-ecp-features.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/portal/applications/using-advanced-ecp-features.md:177]: Done. How should we handle other instances of "persist-ssd," such as the following bullet in the topic Understanding the Dashboard: Capacity of persist-SSD storage?

File: docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (Jianghai-Quantil) [reply docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md]: The linked content is the content you'll see when clicking the link "public network interfaces of a pod replica". Same change needs to be made on the linked content to avoid simply saying "public network".

You can check the current linked content here https://github.com/mileweb/docs-content-ecp/blob/marc-dev/docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md#exposing-via-the-public-network

File: docs/portal/applications/configuring-applications.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (Jianghai-Quantil) [reply docs/portal/applications/configuring-applications.md]: There is an unexpected word which shall be removed. Everything else is fine.
image

File: docs/portal/applications/configuring-applications.md (hunk 22 above)
[the remark this answers is not in the document]
Comment (Jianghai-Quantil) [reply docs/portal/applications/configuring-applications.md]: There is a mispelling on the link to "Using Storage Resources".
"stortage" shall be corrected to "storage"

File: docs/portal/applications/using-advanced-ecp-features.md (hunk 1 above)
Comment (Jianghai-Quantil) [reply docs/portal/applications/using-advanced-ecp-features.md:177]: Please remove "Capacity of persist-SSD storage" from "Understanding the Dashboard"

File: docs/portal/applications/updating-an-application.md (hunk 2 above)
Comment (Jianghai-Quantil) [reply docs/portal/applications/updating-an-application.md:1]: I'd suggest rewording step 3 and step 4 to make it clearer that these 2 steps represent 2 different approaches to update an application. Something like:

Step3: To update an application by editting the YAML file directly, ...
Step4: To quickly update an application without having to touch the YAML file, ...

image

File: docs/portal/applications/updating-an-application.md (hunk 3 above)
Comment (Jianghai-Quantil) [reply docs/portal/applications/updating-an-application.md]: I do see the change. Can you double check?
image

File: docs/portal/applications/updating-an-application.md (hunk 6 above)
Comment (Jianghai-Quantil) [reply docs/portal/applications/updating-an-application.md:12]: The preview of this MD file doesn't look correct to me. Can you check the preview on GitHub as well? https://github.com/mileweb/docs-content-ecp/blob/marc-dev/docs/portal/applications/updating-an-application.md

On my Chrome browser, only "Updating an Application" is correctly rendered. The "#" characters on the other headings are not interpreted. The last 2 headings are not even displayed in their own lines. Besides, please replace Pop with PoP in the heading "Applications Distributed per Pop", and please change the heading "Setting the Container" to "Setting Container Image".
image
image

File: docs/portal/applications/updating-an-application.md (hunk 9 above)
Comment (Jianghai-Quantil) [reply docs/portal/applications/updating-an-application.md]: I confirm the text is correct now. Find a new issue. The screenshot below the sentence doesnot exist.
image

File: docs/portal/applications/adding-and-deploying-applications.md (hunk 11 above)
Comment (Jianghai-Quantil) [reply docs/portal/applications/adding-and-deploying-applications.md:1]: OK, please make further comment if any updates.

File: apispec/openapi.json [hunk 24]
```diff
@@ -479,7 +479,8 @@
           {
             "name": "On-Behalf-Of",
             "in": "header",
-            "description": "If you are a reseller, you can use this header to specify the ID of a child customer and make API call on their behalf.",
+            "description": "If you are a reseller, you can use this header to specify the ID of a child customer and 
+            s on their behalf.",
```
[the remark this answers is not in the document]
Comment (Jianghai-Quantil) [reply apispec/openapi.json:483]: As commented above, can you undo the change to this file so that there won't be any sync issue in the future?

File: docs/portal/applications/viewing-application-information.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (Jianghai-Quantil) [reply docs/portal/applications/viewing-application-information.md:14]: Please give further comment if any updates. Let's make it complete before approving the PR and merging into master branch.

File: docs/portal/applications/updating-an-application.md (hunk 9 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md]: Fixed

File: docs/portal/applications/using-advanced-ecp-features.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/portal/applications/using-advanced-ecp-features.md:177]: Done.

File: docs/portal/applications/viewing-application-information.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/viewing-application-information.md:14]: I replaced screen shots with high-res versions.

File: docs/portal/applications/configuring-applications.md (hunk 22 above)
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/configuring-applications.md]: Fixed.

File: docs/portal/applications/configuring-applications.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/configuring-applications.md]: Fixed

File: docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md]: Fixed.

File: docs/portal/applications/updating-an-application.md (hunk 2 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:1]: Fixed

File: docs/portal/applications/updating-an-application.md (hunk 3 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md]: Fixed by replacing the sentences "The following procedure describes how to update applications from the Applications page. You can also update applications from the application information page." with "The following procedure describes how to update applications."

File: docs/portal/applications/updating-an-application.md (hunk 6 above)
Comment (marcfishelquantil) [reply docs/portal/applications/updating-an-application.md:12]: Fixed.

File: docs/portal/applications/adding-and-deploying-applications.md (hunk 11 above)
Comment (marcfishelquantil) [reply docs/portal/applications/adding-and-deploying-applications.md:1]: Updated hierarchy (routes and paths) to address this issue.

File: docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (Jianghai-Quantil) [reply docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md]: The link behind "public network interfaces of a pod replica" is broken. Please check.
Please change the table heading from "Fields for Exposing Application via the Public Network" to "Fields for Exposing Application via the Public Network Interfaces of a Pod Replica?" Please do include "Replica" at the end.
